cli/app.go

```diff
@@ -1141,9 +1141,9 @@ viam module upload --version "0.1.0" --platform "linux/amd64" packaged-module.ta
 					Action: UploadModuleAction,
 				},
 				{
-					Name:   "build",
-					Hidden: true,
-					Usage: `build your module on different operating systems and cpu architectures via cloud runners.
+					Name:  "build",
+					Usage: "build your module for different architectures using cloud runners",
+					UsageText: `Build your module on different operating systems and cpu architectures via cloud runners.
 Uses the "build" section of your meta.json.
```

Comment:
[nit] we should probably edit this msg to indicate that the user needs to insert this at the top level (because it is not populated by default)

Ex:
```
{
"module_id":...
"description":...
"build": {
   "setup": "setup.sh",                    // optional - command to install your build dependencies
   "build": "make module.tar.gz",          // command that will build your module
   "path" : "module.tar.gz",               // optional - path to your built module
                                           // (passed to the 'viam module upload' command)
   "arch" : ["linux/amd64", "linux/arm64"] // architectures to build for
}
}
```

Reply:
updated docstring